buildpipeline/pipeline.json

```diff
@@ -545,7 +545,7 @@
           "Name": "DotNet-CoreFx-Trusted-Linux",
           "Parameters": {
             "PB_DockerTag": "opensuse421_prereqs_v3",
-            "PB_TargetQueue": "suse.421.amd64"
```

Comment:
I specifically left it alone because some folks were concerned about this breaking other dependent builds.  Since I couldn't get a specific answer the incremental step of just not running the tests is still a step in the right direction.  I'm happy to make a PR that does what you're asking but we should make sure concerns are addressed first.